lesson_07/objects/objects_app/src/test/java/com/codedifferently/lesson7/ChukwumaIbezimTest/BankAccountTest.java

```diff
@@ -0,0 +1,86 @@
+package com.codedifferently.lesson7.ChukwumaIbezimTest;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import com.codedifferently.lesson7.chukwumaibezim.*;
+import org.junit.jupiter.api.Test;
+
+public class BankAccountTest {
+
+  @Test
+  public void testDeposit() {
+    BankAccount account =
+        new BankAccount("Jermaine Cole", 100.0, "2024-03-03", "CHECKING", 1843456889);
+    account.deposit(50.0);
+    assertEquals(150.0, account.getBalance(), 0.0);
```

Comment:
What's the third parameter for? I expected it to be a String message describing what the assertion is expected to confirm.

Reply:
Shows the date the account was opened